content/kubernetes/active-active/_index.md

```diff
@@ -92,7 +92,7 @@ For a full list of fields and options, see the [RERC API reference](https://gith
 * The `<rec-name>/<rec-namespace>` value must be unique for each RERC resource. (RED-96302)
 * Only global database options are supported, no support for specifying configuration per location.
 * No support for migration from old (`crdb-cli`) Active-Active database method to new Active-Active controller.
-* No support for multiple participating clusters on the same Kubernetes cluster.
+* No support for REAADB with participating clusters that reside on the same K8s cluster.
```

Comment:
reading it again it might confuse someone that can think on the local participating cluster as the one that is not supported also...
so to clarify this:
"""
No support for REAADB with two or more participating clusters that reside on the same K8s cluster.
"""